cmake/sysbuild/suit_utilities.cmake

```
@@ -108,3 +108,20 @@ function(suit_create_cache_partition args output_file partition_num recovery)
                  "${output_file_name}type=cache;${output_file_name}partition=${partition_num};")
   endif()
 endfunction()
+
+# Usage:
+#   suit_add_merge_hex_file(FILES <files> [DEPENDENCIES <dependencies>]
+#
+# Add files which should be merged into the uicr_merged.hex output file, respecting any
+# dependencies that need to be generated before hand. This will overwrite existing data if it is
+# present in other files
+function(suit_add_merge_hex_file)
+  cmake_parse_arguments(VAR "" "" "FILES;DEPENDENCIES" ${ARGN})
```

better to start using `arg` as prefix.
See: https://gitlab.kitware.com/cmake/cmake/-/issues/25773, and https://gitlab.kitware.com/cmake/cmake/-/merge_requests/9788/diffs

```suggestion
  cmake_parse_arguments(arg "" "" "FILES;DEPENDENCIES" ${ARGN})
```

---

Fixed